### PR TITLE
Update Code Syntax in Alerting in Astronomer Software

### PR DESCRIPTION
**Pull request by kwcanuck**
In this [Slack conversation](https://astronomer.slack.com/archives/C015V2JFKT5/p1675211068787069), @afastronomer requested that instances of `platform:` be updated to `platform: |`.

### CODE REVIEW

**Review by afastronomer — CHANGES_REQUESTED** [review]
Fix is required only for Prometheus (additionalAlerts), Alertmanager (receivers) config has to be unchanged.

**Comment by kwcanuck** [comment]
@afastronomer the docs have been updated as you requested. Please review and unblock the PR. Thanks!

**Review by afastronomer — APPROVED** [review]

**Comment by kwcanuck** [comment]
@jwitz a quick review for you - I hope!

**Review by jwitz — APPROVED** [review]

File: software/platform-alerts.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For example, adding the following receiver to `receivers.platformCritical` would
 alertmanager:
   receivers:
     # Configs for platform alerts
-    platform:
+    platform:
```
Comment (jwitz) [inline software/platform-alerts.md:49]:
```suggestion
    platform: |
```
Do we need one here as well?

Comment (kwcanuck) [reply software/platform-alerts.md:49]: @jwitz as Artem indicated "Alertmanager (receivers) config has to be unchanged".

**Pull request merged**